public/pages/Policies/components/PolicyControls/__snapshots__/PolicyControls.test.tsx.snap

```diff
@@ -101,3 +101,105 @@ exports[`<PolicyControls /> spec renders the component 1`] = `
   </div>
 </div>
 `;
+
```

Comment:
Not sure what happened here, seems like this duplicated above?

Reply:
Cleaned up